components/menu/menu.js

```diff
@@ -67,6 +67,10 @@ class Menu extends HierarchicalViewMixin(LitElement) {
 		this._labelChanged();
 
 		this._onMenuItemsChanged();
+		const slot = this.shadowRoot.querySelector('slot');
+		slot.addEventListener('dom-change', () => {
```

Comment:
You're interested in finding out when the distributed nodes change? I think `dom-change` is a Polymer thing... does the `slotchange` standard event work?

Reply:
Yup - looks like `slotchange` will work!